Formula/jupyterlab.rb

```diff
@@ -405,8 +407,15 @@ def caveats
     system bin/"jupyter-nbconvert", "nbconvert.ipynb", "--to", "html"
     assert_predicate testpath/"nbconvert.html", :exist?, "Failed to export HTML"
 
-    assert_match "-F _jupyter",
-      shell_output("source #{bash_completion}/jupyter && complete -p jupyter")
+    on_macos do
+      assert_match "-F _jupyter",
+        shell_output("source #{bash_completion}/jupyter && complete -p jupyter")
+    end
+
+    on_linux do
+      assert_match "-F _jupyter",
+        shell_output("bash -c \"source #{bash_completion}/jupyter && complete -p jupyter\"")
+    end
```

Comment:
```suggestion
    assert_match "-F _jupyter",
                 shell_output("bash -c 'source #{bash_completion}/jupyter && complete -p jupyter'")
```
Let's just use `bash` on both macOS and Linux.